Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1824,6 +1850,17 @@ extension BrowserViewController: ReaderModeDelegate, UIPopoverPresentationContro
     }
 }
 
+// MARK: - UIPopoverPresentationControllerDelegate
```

Comment:
I added some `//MARK: -` comments to ease navigation in the code. There is a lot of code in BrowserViewController and even though it is splitted in several extensions, it it hard to navigate from one extension to another with the Xcode jump bar without these marks.